eslint-plugin-javascript/src/rules/use-app-alternatives.ts

```diff
@@ -0,0 +1,103 @@
+import { getFullFunctionName, parseOptions, PLUGIN_DOCS_URL } from '../utils';
```

Comment:
This rule should be configurable by multiple configs.

redux config
	useDispatch => useAppDispatch
commie config
	myStuff => ourStuff

Both configs should work.

Reply:
Currently there is **no** way to achieve this.
There was a proposal to add options merging but it was closed:
https://github.com/eslint/eslint/issues/9192
Also the ESLint settings can't help us because they are only merged if the subkeys are different but they are not merged if the keys are the same, instead the last one takes precedence:
```js
export default {
  {
    "settings": {
      "cats": {
        "Kozunak": 1,
        "CatNameThatExistsTwice": 10
      }
    }
    // config stuff here
  },
  {
    // other config
     "settings": {
      "cats": {
        "Roko": 2,
        "CatNameIsUsedTwice": 20
      }
    }
  }
}
```

Will result in:
```js
settings: {
  "cats": {
    "Kozunak": 1,
    "Roko": 2,
    "CatNameIsUsedTwice"
  }
}
```